<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MiscellaneousCustomTextEdit</class>
 <widget class="MiscellaneousBase" name="MiscellaneousCustomTextEdit">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>803</width>
    <height>711</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MiscellaneousCustomTextEdit</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="lbTitle">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>40</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>12</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Custom TextEdit For Right Click Menu</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <property name="spacing">
      <number>6</number>
     </property>
     <property name="leftMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="QLabel" name="lbAppendText">
       <property name="text">
        <string>追加文本：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="leAppendText"/>
     </item>
     <item>
      <widget class="QPushButton" name="btnAppendText">
       <property name="text">
        <string>追加文本</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnTestEntry">
       <property name="text">
        <string>测试入口</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="JcbTextEdit" name="teCustom">
       <property name="styleSheet">
        <string notr="true">QTextEdit
{
	background-color: lightgreen;
}</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QTextEdit" name="teOriginal"/>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>MiscellaneousBase</class>
   <extends>QWidget</extends>
   <header>miscellaneousbase.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>JcbTextEdit</class>
   <extends>QTextEdit</extends>
   <header>JcbTextEdit.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
